Source/interface_example.py

```diff
@@ -0,0 +1,63 @@
+#
+# License: See LICENSE file
+# GitHub: https://github.com/Baekalfen/PyBoy
+#
+
+from pprint import pprint
+from pyboy import PyBoy
+
+# pyboy = PyBoy("ROMs/POKEMON BLUE.gb", window_type="headless", window_scale=1) # For unattended use, for example machine learning
+pyboy = PyBoy("ROMs/POKEMON BLUE.gb", window_type="SDL2", window_scale=1)
+pyboy.set_emulation_speed(False, 0)
+
+for n in range(1000): # Move ahead the desired number of frames.
+    pyboy.tick()
+
+tile_map = pyboy.get_window_tile_map() # Get the TileView object for the window.
+
+# The following prints out the indices of each tile in the window -- excluding sprites and the background!
+print(tile_map)
```

Comment:
This output does not render anymore after the changes to TileMap's behavior: `__str__` fails because `get_tile_matrix()` returns identifiers instead of tiles. I'll attach a change that fixes this in `tilemap.py`, though.